tests/framework/pvc.go

```diff
@@ -109,7 +109,8 @@ func (f *Framework) VerifyTargetPVCContent(namespace *k8sv1.Namespace, pvc *k8sv
 	if err != nil {
 		return false, err
 	}
-	return strings.Compare(expectedData, output) == 0, nil
+	outputWithoutNullBytes := strings.Trim(output, "\x00")
```

Comment:
My problem with this is that we have to store all the \x00 in memory after getting it from the pod, and then we trim it before testing. So if the PVC is large, we will have to store a large number of bytes for no reason. Instead since we know how many bytes the actual data is, we should just grab the actual number of bytes we are interested in, and compare that. Everywhere else in the testing framework we have switched over to VerifyTargetPVCContentMD5 and we pass in the number of bytes we want to check.

Second, it doesn't look like the modified tests actually are calling VerifyTargetContent at all,

Reply:
Switched the remaining use, too.